build_tools/buildkite/presubmit.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Note: this runs on "security: submitted" agents, since this pipeline itself is
+# submitted code and it fetches the scripts from the main branch.
+
+steps:
+  # TODO: Is there a better emoji here? Google logo (would have to add to
+  # Buildkite)?
+  - label: ":guardsman: :pouting_cat: :raised_hand: Check CLA"
+    agents:
+      queue: "orchestration"
+      security: "submitted"
+    # We're not checking out from the PR and besides saving time, we really
+    # don't want to because we don't want unsubmitted code on these executors at
+    # all.
+    plugins:
+      - thedyrt/skip-checkout#0870506e0b5d4becc164c6e64dbc331938f0bdcf:
+          cd: /tmp/
+    commands: |
+      # TODO: Switch to main branch
+      curl \
+        https://raw.githubusercontent.com/GMNGeoffrey/iree/whole-new-world/build_tools/buildkite/check_cla.py \
+        | python3 - ${BUILDKITE_COMMIT}
+
+  - wait
```

Comment:
Can we bypass the CLA check to run presubmits if someone trusted adds a label / triggers a pipeline manually? Some corporate and individual CLAs can take a while to process.